scripts/filter_fdb_entries.py

```diff
@@ -8,33 +8,67 @@
 import traceback
 import time
 
+from ipaddress import ip_address, ip_network, ip_interface
 from collections import defaultdict
 
-def get_arp_entries_map(filename):
+def get_vlan_cidr_map(filename):
+    """
+        Generate Vlan CIDR information from Config DB file
+
+        fdb entries could be contaminated with foreigh Vlan entries as seen in the case of 
+        FTOS fast conversion. SONiC Vlan CIDR configuration will be used to filter out
+        those invalid Vlan entries out.
+
+        Args:
+            filename(str): Config DB data file
+
+        Returns:
+            vlan_cidr(dict) map of Vlan CIDR configuration for SONiC device
+    """
+    with open(filename, 'r') as fp:
+        config_db_entries = json.load(fp)
+
+    vlan_cidr = defaultdict()
+    if "VLAN_INTERFACE" in config_db_entries.keys() and "VLAN" in config_db_entries.keys():
+        for vlan_key in config_db_entries["VLAN_INTERFACE"].keys():
+            vlan, cidr = tuple(vlan_key.split('|'))
```

Comment:
There could be entry without cidr in config_db. You may want to skip such entries or it may fail. Can you also put those entries in test_vector and verify. 

            "VLAN_INTERFACE": {
                "Vlan1000": {}
            },

Reply:
Good catch. did not know such entries exists. @yxieca FYI... we might want to hold off on .81.

Reply:
Actually, these entries exists in 201911 and master. 201811 doesn't have such entries FYI

Reply:
@prsunny, is this a bug in these branches! I did not recall seeing them, however when you alerted me, it happened that I had master image on my DUT and I saw it! I created a fix for it. No, I am inclined to know why they exist on master and on 201911. Sounds like a bug there that needs to be addressed.

Reply:
it is not a bug, but a feature introduced as part of VRF in 201911 and above

Reply:
@tahmed-dev is above comment has been addressed ?

Reply:
@abdosi, yes it is fixed on master. please cherry-pick this [PR](https://github.com/Azure/sonic-utilities/pull/976) as well.